helm/hpcc/templates/_helpers.tpl

```diff
@@ -2058,3 +2058,31 @@ spec:
 {{- end -}}
 {{- end -}}
 
+{{ define "hpcc.generateWarnings" }}
```

Comment:
minor: why was the whitespace chomping removed here (in commit 5) ?

All our helper functions have whitespace chomping here to avoid introducing new lines each time they are called

Reply:
@shamser - fixing this (adding the chomping back) caused problems?

My guess is that it's because on line 2068:
```
{{- if $ctx.warnings -}}
```

that shouldn't chomp the newline at the end. It should be:
```
{{- if $ctx.warnings }}
```

I suspect spurious newline(s) introduced on the {{ define .. }} were working around the chomp here.

Reply:
Tested. I made the change above, and added back the chomping to the generateWarnings, printWarnings and getWarning define's and it generate correctly.